clerk/sessions.go

```diff
@@ -15,6 +15,26 @@ type Session struct {
 	AbandonAt    int64  `json:"abandon_at"`
 }
 
+type ListAllSessionsParams struct {
+	Limit    *int
+	Offset   *int
+	ClientID *string
+	UserID   *string
+	Status   *SessionStatus
+}
+
+type SessionStatus string
+
+const (
+	STATUS_ABANDONED SessionStatus = "abandoned"
+	STATUS_ACTIVE    SessionStatus = "active"
+	STATUS_ENDED     SessionStatus = "ended"
+	STATUS_EXPIRED   SessionStatus = "expired"
+	STATUS_REMOVED   SessionStatus = "removed"
+	STATUS_REPLACED  SessionStatus = "replaced"
+	STATUS_REVOKED   SessionStatus = "revoked"
+)
```

Comment:
:wrench: Consider prefixing these constants with `SESSION_`. Since everything's inside the Clerk package, if we ever want to add similar constants to other resources, there's going to be a naming collision.

Also, let's write them in camel case, maybe it's more Go idiomatic.

```go
// This is for sessions
const SessionStatusAbandoned SessionStatus = "abandoned"

// This is for some other resource
const OtherResourceStatusAbandoned = "abandoned"

// They are both exported by the clerk package
clerk.OtherResourceStatusAbandoned
clerk.SessionStatusAbandoned
```

Reply:
Sounds good! I never know what to do with constants... thanks!